secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+import os
+import time
+from typing import Iterator, Optional, Tuple, Union
+import uuid
+
+from google.api_core import exceptions, retry
+from google.cloud import secretmanager, secretmanager_v1
+import pytest
+
+from regional_samples import create_regional_secret
+from regional_samples import delete_regional_secret
+from regional_samples import get_regional_secret
+from regional_samples import regional_quickstart
+from regional_samples import update_regional_secret
+
+
+@pytest.fixture()
+def location_id() -> str:
+    return "us-east5"
+
+
+@pytest.fixture()
+def regional_client(location_id: str) -> secretmanager_v1.SecretManagerServiceClient:
+    api_endpoint = f"secretmanager.{location_id}.rep.googleapis.com"
+    return secretmanager_v1.SecretManagerServiceClient(
+        client_options={"api_endpoint": api_endpoint}
+    )
+
+
+@pytest.fixture()
+def project_id() -> str:
+    return os.environ["GOOGLE_CLOUD_PROJECT"]
+
+
+@pytest.fixture()
+def iam_user() -> str:
+    return "serviceAccount:" + os.environ["GCLOUD_SECRETS_SERVICE_ACCOUNT"]
+
+
+@pytest.fixture()
+def ttl() -> Optional[str]:
+    return "300s"
+
+
+@retry.Retry()
+def retry_client_create_regional_secret(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    request: Optional[Union[secretmanager_v1.CreateSecretRequest, dict]],
+) -> secretmanager_v1.Secret:
+    # Retry to avoid 503 error & flaky issues
+    return regional_client.create_secret(request=request)
+
+
+@retry.Retry()
+def retry_client_delete_regional_secret(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    request: Optional[Union[secretmanager_v1.DeleteSecretRequest, dict]],
+) -> None:
+    # Retry to avoid 503 error & flaky issues
+    return regional_client.delete_secret(request=request)
+
+
+@retry.Retry()
+def retry_client_access_regional_secret_version(
+    regional_client: secretmanager_v1.SecretManagerServiceClient,
+    request: Optional[Union[secretmanager_v1.AccessSecretVersionRequest, dict]],
+) -> secretmanager_v1.AccessSecretVersionResponse:
+    # Retry to avoid 503 error & flaky issues
+    return regional_client.access_secret_version(request=request)
+
+
+@pytest.fixture()
+def secret_id(
+    regional_client: secretmanager.SecretManagerServiceClient,
```

Comment:
Is this supposed to be different?

Reply:
@dandhlee I am not sure what you mean here. Different from ?

Reply:
Sorry - this is `secretmanager` vs `secretmanager_v1`, any reason why it's using a different version of the client?

Reply:
Thanks. Fixed